src/eip7594/eip7594.c

```diff
@@ -648,10 +667,8 @@ static C_KZG_RET compute_commitment_to_aggregated_interpolation_poly(
         if (ret != C_KZG_OK) goto out;
 
         /* Now divide by the coset shift factor */
-        uint64_t pos = reverse_bits_limited(CELLS_PER_EXT_BLOB, i);
-        fr_t inv_coset_factor;
-        blst_fr_eucl_inverse(&inv_coset_factor, &s->roots_of_unity[pos]);
-        shift_poly(column_interpolation_poly, FIELD_ELEMENTS_PER_CELL, &inv_coset_factor);
+        uint64_t pos = -CELL_INDICES_RBL[i] % FIELD_ELEMENTS_PER_EXT_BLOB;
```

Comment:
Please see my other comment about `coset_shift_for_cell()`. This change can be confusing IMO. Not sure what `-CELL_INDICES_RBL[i]` is supposed to be, let alone comboed the modulus operator.

Reply:
That's equivalent to the following:

```c
FIELD_ELEMENTS_PER_EXT_BLOB - CELL_INDICES_RBL[i] % FIELD_ELEMENTS_PER_EXT_BLOB
```

It's the formula I came up with to get the inverse root of unity.

Reply:
You probably meant

`(FIELD_ELEMENTS_PER_EXT_BLOB - CELL_INDICES_RBL[i]) % FIELD_ELEMENTS_PER_EXT_BLOB`

Reply:
Yes, thank you.

Reply:
I still think it is confusing. Maybe a more detailed comment would help? 😅

Reply:
I've added some slightly better comments. Let me know if those are fine.